docs/notebooks/mango-python-alignment.ipynb

```diff
@@ -0,0 +1 @@
+../../example-files/notebooks/mango-python-alignment.ipynb
```

Comment:
We should check if readthedocs is able to access files external to the docs folder

Reply:
> reading sources... [100%] installation/source

When you "try to build" above, was that you running `make html` in the docs folder?
And when you say the sidebar is missing are you saying that:
![image](https://user-images.githubusercontent.com/7423296/44533067-ae200380-a6c2-11e8-864c-d510a73a99dc.png)
is missing?


( I am out of town until Fri evening, I look into further then)

Reply:
Yes, when I ran `make html` it failed. I correctly installed the npshpinx/convert dependencies. Is there another command I need to run?

You can refer to the current docs https://bdg-mango.readthedocs.io/en/latest/ to see how the index should look.

Reply:
Upgrading to 
sphinx==1.7.7 and nbsphinx=1.3.4
appears to fix the problem with the sidebar.
Let me know if you still have trouble with `make html`

Reply:
The build is still not working. See comment above regarding missing jars.